packages/api-client/src/components/AddressBar/AddressBarServer.vue

```diff
@@ -84,7 +89,7 @@ const serverUrlWithoutTrailingSlash = computed(() => {
 
       <template v-if="showDropdownLabels">
         <ScalarDropdownDivider />
-        <div class="text-xxs text-c-2 ml-8">Collection Servers</div>
+        <div class="text-xxs text-c-2 px-2.5 py-1">Collection</div>
```

Comment:
I'm open to adding a `ScalarDropdownTitle` or something to components if we want